website/versioned_docs/version-latest/experiments/remote_taskfiles.mdx

```diff
@@ -104,21 +104,33 @@ Whenever you run a remote Taskfile, the latest copy will be downloaded from the
 internet and cached locally. If for whatever reason, you lose access to the
 internet, you will still be able to run your tasks by specifying the `--offline`
 flag. This will tell Task to use the latest cached version of the file instead
-of trying to download it. You are able to use the `--download` flag to update
-the cached version of the remote files without running any tasks. You are able
-to use the `--clear-cache` flag to clear all cached version of the remote files
-without running any tasks.
+of trying to download it.
 
-By default, Task will timeout requests to download remote files after 10 seconds
-and look for a cached copy instead. This timeout can be configured by setting
-the `--timeout` flag and specifying a duration. For example, `--timeout 5s` will
-set the timeout to 5 seconds.
+The flags to affect the caching behaviour are as follows:
+
+- `--download` (boolean): update the cached version of the remote files without
+  running any tasks.
+- `--clear-cache` (boolean): clear all cached version of the remote files
+  without running any tasks.
+- `--remote-cache-ttl` (duration): override the default time-to-live (TTL) of
+  remote Taskfiles in the cache, when there is a cached Taskfile present and
+  deciding whether to use the cached copy or download a newer version.
+
+By default, Task requests to download remote files will time out after 10
+seconds and look for a cached copy instead. This timeout can be configured by
+setting the `--timeout` flag and specifying a duration. For example,
+`--timeout 5s` will set the timeout to 5 seconds.
 
 By default, the cache is stored in the Task temp directory, represented by the
 `TASK_TEMP_DIR` [environment variable](../reference/environment.mdx) You can
 override the location of the cache by setting the `TASK_REMOTE_DIR` environment
 variable. This way, you can share the cache between different projects.
 
+The default TTL for remote Taskfiles in the cache is 24 hours. This can be
+overridden by the `--remote-cache-ttl` flag outlined above. The value must be
+specified as a duration, e.g. `30s`, `15m`, `4h`. More information on how
+[durations are parsed is available here](https://pkg.go.dev/time#ParseDuration).
+
 {/* prettier-ignore-start */}
 [enabling-experiments]: ./experiments.mdx#enabling-experiments
 [man-in-the-middle-attacks]: https://en.wikipedia.org/wiki/Man-in-the-middle_attack
```

Comment:
This should be done only in `webiste/docs` and reverted from this file.
`webiste/versioned_docs` is a generated folder and will be overriden

Reply:
Noted, thanks! 
When I edited the other file and ran a preview on my local, it didn't show any of the changes. 
I'll move them over when I get back to my desk.

Reply:
Yeah it's because we have two version of the doc : 
- The stable's one (the one you see when you go to taskfile.dev, or when running the server locally. It point to the latest release
- The "next's one" which point to main

To access to the "next's one", you need to click in the header bar : 
![image](https://github.com/go-task/task/assets/9110126/de95076e-742d-4729-b4ac-6be0d54b9873)

Reply:
e309b73f455d88cd133f6730e9b09ed3fd0c5063 will resolve.